docs/src/content/docs/for-administrators/configuring-extra-files.md

```diff
@@ -77,6 +79,10 @@ For this to work, you need to configure a bucket policy like this:
 
 Consult the documentation of your particular S3 provider on how to configure bucket policies.
 
+:::note
```

Comment:
can you move this note to the beginning of the block? (under the "Set S3 policy")